0018-connector-risk-mitigations/0018-connector-risk-mitigations.md

```diff
@@ -17,22 +17,25 @@ The main risk connectors face in Interledger is being unable to fulfill the inco
 Possible mitigations include:
 
 * **Connecting Reliable Ledgers** - Connectors should choose the ledgers they connect in part based on the reliability and speed of fulfillment notifications and the time the ledgers require to process fulfillments once they are submitted
-* **Adjusting Transfer Expiry Window** - Connectors determine the window of time they require between when the incoming and outgoing transfers expire. Connectors should set this window such that they are confident that they will be able to deliver the fulfillment in time even if the outgoing transfer is executed at the last possible moment.
+* **Adjusting Transfer Expiry Window** - Connectors determine the window of time they require between when the incoming and outgoing transfers expire. Connectors should set this window such that they are confident that they will be able to deliver the fulfillment in time even if the outgoing transfer is executed at the last possible moment. TODO: Not sure if still relevant because we are not using HTLCs
```

Comment:
Is this still relevant? We're not using HTLA/Cs right, so I think it's not

Reply:
@emschwartz

Reply:
`Connecting Reliable Ledgers` is not relevant anymore.

However, I'd say `Adjusting Transfer Expiry Window` is. While this is no longer enforced by the ledger, it is still part of the protocol rules. A connector does still need to adjust the expiry in outgoing prepare packets to give themselves time to process and pass on the fulfill.